src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -647,18 +647,20 @@ studio.search.facets:
     field: lastEditedOn
     date: true
     ranges:
-      - label: older
-        to: now/d-365d
-      - label: within-a-year
-        from: now/d-365d
-      - label: within-6-months
-        from: now/d-180d
-      - label: within-a-month
-        from: now/d-30d
-      - label: within-a-week
+      - label: in-last-day
```

Comment:
Change to say today. Need to update the other related PRs.

Reply:
fortunately there's a "today" translation in search bundle, so this is the only file that needs to change